packages/runtime/schema.json

```diff
@@ -733,6 +733,76 @@
             }
           },
           "additionalProperties": false
+        },
+        "formatters": {
+          "type": "object",
+          "properties": {
+            "bindings": {
+              "type": "object",
+              "properties": {
+                "path": {
+                  "type": "string",
+                  "resolvePath": true
+                }
+              },
+              "additionalProperties": false
+            },
+            "level": {
+              "type": "object",
+              "properties": {
+                "path": {
+                  "type": "string",
+                  "resolvePath": true
+                }
+              },
+              "additionalProperties": false
+            }
+          },
+          "additionalProperties": false
+        },
+        "timestamp": {
+          "anyOf": [
+            {
+              "type": "boolean"
+            },
+            {
+              "enum": [
+                "epochTime",
+                "unixTime",
+                "nullTime",
+                "isoTime"
+              ]
+            },
+            {
+              "type": "object",
+              "properties": {
+                "path": {
+                  "type": "string",
+                  "resolvePath": true
+                }
+              },
+              "additionalProperties": false
+            }
+          ]
+        },
+        "redact": {
+          "type": "object",
+          "properties": {
+            "paths": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "censor": {
+              "type": "string",
+              "default": "[redacted]"
+            }
+          },
+          "required": [
+            "paths"
+          ],
+          "additionalProperties": false
```

Comment:
I don't see the matching code change for this.

Reply:
I've reverted this